### ISSUE TITLE
[Bug] My device (client) gets dis-/reconnected regularly or not tracked well

### ISSUE THREAD

**Issue by Vaskivskyi** [Vaskivskyi/ha-asusrouter#792]
## The new general issue to report about tracking issues with some of the clients (connected devices)

You can report or discuss ONLY issues with clients tracking here. Or, you can report that your client tracking got fixed in some integration version (even if only for a single client, but not for all of them).

### Here is what you need to say if you want your issue to be checked faster:

- Your router model and FW version (you can avoid it only if you have already specified it in THIS same issue & it didn't change)
- Integration version where it does not work (e.g. `0.29.0` or the last commit hash if you used the dev branch directly). There is no `latest` version
- Client info that got troubles (including `it connects/disconnects regularly` and so on). Please, specify every device that has issues (even if it's `all of them` for you - this sentence sure sounds better, but does not help in troubleshooting.

#### What data to provide per-client

- Short client info (make, model, OS / FW)
- Client connection type (e.g. `wired`, `2.4 GHz`, `5 GHz`, `6 GHz`, `LACP` / `link aggregation`)
- Whether the client is connected `directly` to the main router or with any additional device in-between (e.g. `aimesh node`, `access point`, `media bridge`, `switch` and so on) and how (e.g. `through an aimesh node in 5 GHz backhaul`)

### Example of the issue report

> Hello, my Synology NAS gets connected and disconnected regularly. All the other clients seem to work fine
> My router is GT-AX11000 Pro with stock 3.0.0.4.388_24198-g380c137 FW
> Synology DS923+ with DSM 7.2.1. Connected directly to the router but uses link aggregation
> I am on version 0.29.0 of the integration

Another example

> Looks like with 0.29.0 I am getting issues with iOS devices. RT-AX88U with AiMesh of 2 RT-AX92U
> It happens for both iPad Air and iPhone 15 Pro with iOS 17.2 and does not seem to matter whether they are connected directly to the router or to an AiMesh node

### Your device model

GT-AX11000 Pro

### Firmware type

Stock

### Firmware version

3.0.0.4.388_24198-g380c137

### Integration version with the issue

0.29.0

### Method of the integration installation

HACS

### What version and type of Home Assistant installation do you use

Core 2024.2.0

**Comment by h3llrais3r** [comment]
Asusrouter integration: `0.28.0`
Asusrouter: `RT-AX58U`
Router firmware: `3.0.0.4.388_23925-g269b2a6`
POE switch: `TP-Link TLSG108PE`
Access point: `TP-Link Omada EAP225_OUTDOOR`

I mainly have the connect/disconnect isues with devices connected via either switch or access point.
- Camera devices connected via poe switch: `HIKVISION DS-2CD2345FWD-I`
- Shelly devices connected via access point: `Shelly Plus 1` and `Shelly Plus 2pm`
- Sonoff device: `NSPanel`

All of them are connected via 2.4GHz network.
Router setup has both 2.4GHZ and 5GHz network configured with same SSID and password, but on different band.

Another thing I noticed (not sure it's related) is that devices, connected wirelessly via the access point, are seen as wired connections... (maybe because the access point itself is wired?)

**Comment by Vaskivskyi** [comment]
@h3llrais3r, I will try to find some switch and AP to test the behaviour. Hopefully, this is more `type of connection` related, than exact model related.

Meanwhile, can you please check that the data is shown the same way when you open the `Network Map` page in Web UI and the `AiMesh` page? I have added a short description to the original issue as an explanation. Unfortunately, it seems that different parts of the UI are providing different data and AsusRouter is taking AiMesh data if it's supported

**Comment by milandzuris** [comment]
Router: RT-AX53U FW: 3.0.0.4.386_69061
Integration: 0.29.0
2.4GHz Devices Connected Directly Always Disconnected, Samsung Phones, ESP Smart Devices, i have many Devices 50+ Tracks only 30-35

**Comment by boheme61** [comment]
I haven't pulled my thump out, and analyze the specific device pattern , however today i picked those i remembered as the most frequent "bouncing"  devices. 

1. Wired, MainHome, window laptop( Asus-Pro) i work from all day, and monitoring HA etc(and this, surf etc), it only sleeps when i sleep :) Connected to Router via ( 2m TP-cable) to un-managed GiGABiT(older) switch(5 ports)( 5m TP-cable to Router )
2. Wired, Statdard Laptop, Windows machine, where i have my HA in a WMware, same switch as above (GiGABiT(older) 
( Only the laptops IP(device-tracker) ... ( HA( 7 ) seems stable at the moment, been like that lately, also !)
3. Wired, Phillips Android TV ( Wired to Router( AX68U) via a DL-DGS-1008D switch)( I watched TV all the time writing this
4. Wired, H200 Tapo hub(with 5 devices, 866ghz), connected through same switch above DL-DGS-1008D switch)
* 
5. Wired, ( H-ikea )Ikea Tradfri Hub( 5plugs, 3sensors, 2 light-Drivers,2 shorts-cuts,  Zigbee) same DL-DGS-1008D switch ( Ikea-Integration )
6. Wired, A-qara hub ( 14 sensors, Zigbee ) same DL-DGS-1008D  ( HomeKit Device-Integration )

So My current analyses shows to my "surprise" that 5, 6 and 7 seems rock-solid stable
( Damit while writing this H-Ikea(5) went Away !!!)
( EDIT: as i test i walked down the Hallway, and triggered a motion-sensor turning on the Lights (Ikea Switch), and it came back "Home" ... which explain why it came back ... but apparently quickly ( Went Away again )( The 3 min is the time from motion-detected = lights-on, to turn off again ) ( Most of the devices in H-Ikea have "motion-automations associated with it

1 to 4 , is only device_trackers laptops an tv, which i have not integrated in HA, so these seems somehow "out of the equation" as i don't even has automations etc applied to these

Anyway with this (new findings/analyze) , i back out of the topis , i will report back if i find that it's to hasty a conclusion, but as it looks now, something has improved ( beside the not-integrated Devices )

PS: Also looks like the wifi-plugs in LocalTuya is stable now ( Maybe because i switched to the new Fork , of that integration )

So bottom line is, i really don't have mush to add ( As im sure the not ( in HA ) integrated Devices , would most likely behave different, if there was an entity(device)

![01 03 2024_00 39 29_REC](https://github.com/Vaskivskyi/ha-asusrouter/assets/37634316/9858b5cb-ae10-4f5b-a632-b5917bb308cc)
![29 02 2024_22 53 03_REC](https://github.com/Vaskivskyi/ha-asusrouter/assets/37634316/937291a1-0c72-408e-aed3-bfcc648dd698)
![29 02 2024_22 52 20_REC](https://github.com/Vaskivskyi/ha-asusrouter/assets/37634316/4727ae44-55e6-4d09-9bbe-a4f788417349)

PS: Latest HA, AsusRouter-integration, and firmware in Router

BR Jorgen

**Comment by Korpip** [comment]
I am noticing the same issue.  I have two Poe switches.  One plugged into the main mesh router and the other to a node.

The switch plugged into the node has disconnect and reconnect events repeatedly.
Oddly enough while I had the automation running only one of the Poe cameras connected did the same.  The others didn't.

Also a wifi IP camera did a few connection events.  Which is connected to the same node.

**Comment by milandzuris** [comment]
@Vaskivskyi something new?

**Comment by Vaskivskyi** [comment]
Unfortunately, not yet

**Comment by Korpip** [comment]
Since this is happening for the switches I have them ignored when sending notifications.  The devices connected to the switches are not triggering but if they all do then I know to check to switch.

**Comment by pbuergi** [comment]
> Router: RT-AX53U FW: 3.0.0.4.386_69061 Integration: 0.29.0 2.4GHz Devices Connected Directly Always Disconnected, Samsung Phones, ESP Smart Devices, i have many Devices 50+ Tracks only 30-35

Kinda the sam here:
Also with [AsusWRT Integration](https://www.home-assistant.io/integrations/asuswrt/).
Device: RT-AX95Q (XT8 AX6600)
AP-Mode
FW: 3.0.0.4.388.24688_gf94212b
Integration: 0.34.1
Track-Devices: Samsung Smartphones
Problem: State is always Home or not home, not changing at all. When reloading integration the state is the same as at the beginning of activating the tracking device
Connection: 5ghz
Attribute "Last activity" changes frequently (correct)

**Comment by fmonday** [comment]
Pair of Asus GT-6s in AIMesh mode
FW: stock 3.0.0.4.388_25133-g27bb2a9 (router and node)
Integration: 0.36.0. Both the router and the node have separate entries in the integration- the latter doesn't seem to provide device trackers. 
Devices trying to track: iPhones, laptops, string lights
Connection: 5 GHz-2 channel is dedicated backhaul. 5 GHz-1 is disabled on the node. 2.4 GHz is how all clients connect to the node in question. 
Problem: none of the devices connected to the node show as home. The total connected devices sensor is equal to only the devices connected to the router (wirelessly and wired). The clients connected in the router GUI dashboard and AIMesh pages are correct, as is the Network Map.
